jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -240,6 +253,9 @@ class java_bytecode_languaget:public languaget
   /// It tracks objects that should be reference-equal to each other by mapping
   /// IDs of such objects to symbols that store their values.
   std::unordered_map<std::string, object_creation_referencet> references;
+
+  /// Enables checking whether we should elaborate a method body
+  std::unique_ptr<prefix_filtert> method_in_context;
```

Comment:
:bulb: It is a pair of list of user provided strings, storing as an `optional<prefix_filtert>` seems OK and simplifies the calling code

Reply:
As a unique pointer it makes me the reader assume there is going to be a virtual function or something.